pokemongo_bot/api_wrapper.py

```diff
@@ -8,13 +8,18 @@
 from pgoapi.pgoapi import PGoApi, PGoApiRequest, RpcApi
 from pgoapi.protos.POGOProtos.Networking.Requests_pb2 import RequestType
 
-from human_behaviour import sleep
+from human_behaviour import sleep, gps_noise_rng
 
 class ApiWrapper(PGoApi):
-    def __init__(self):
+    def __init__(self, replicate_gps_noise=False, gps_noise_range=0.00025):
         PGoApi.__init__(self)
         self.useVanillaRequest = False
 
+        if replicate_gps_noise:
+            self.gps_noise_range = gps_noise_range
```

Comment:
1. If you are moving, they can't separate gps noise from your actual position (so it should be ok.)
2. If you are stopping for a long while, they can check if the user is using gaussian distribution or not, but I guess usual gps noises look like gaussian, so they should detect normal user as bot at the same time(as far as the parameters are set correctly)
3. I'm not sure with altitude, checks are needed.
4.I believe drawing new random noise at set_position is quite fine at this point.
5.If anyone is paranoid, he can develop time, position and velocity dependent covariant random noise himself (and share it with us!)